### ISSUE TITLE
Update advanced search page button styles

### ISSUE THREAD

**Issue by RayBB** [internetarchive/openlibrary#8963]
### Problem

We only use `generic-button` in two places in the codebase:
- [Lists page](https://openlibrary.org/lists) - where its CSS styling does not have any effect.
- [Advance search page](https://openlibrary.org/advancedsearch) - which displays a prominent red button. This button's design is inconsistent with the rest of the search pages and with the overall Open Library (OL) aesthetic.
<img width="91" alt="red button" src="https://github.com/internetarchive/openlibrary/assets/921217/7976d778-c6af-4632-adfd-d3af67f2513c">

We have input from the community and the issu eis ready to be assigned, the following actions are proposed:
- [ ] add the `olform` class to the advanced search page
- [ ] Eliminate all instances of the `generic-button` code from the codebase (refer to the screenshot section).
- [ ] Include before/after screenshots in the pull request to illustrate the changes.
- [ ] Remove the `generic-button` from the design library documentation [in the wiki](https://github.com/internetarchive/openlibrary/wiki/Design-Pattern-Library#buttongeneric) to prevent future use.
- [x] Once this is done we can make a separate Issue to evaluate unifying the styles of the search buttons with something like `cta-btn cta-btn--small cta-btn--vanilla` 
  - #9086

<details>
  <summary>Here is what the other search pages look like, and so should this one.</summary>
  
  
<img width="609" alt="image" src="https://github.com/internetarchive/openlibrary/assets/921217/54eb641f-90f0-4935-b2dc-f02bfdd5a409">

  
</details> 

#### Evidence / Screenshot
<img width="860" alt="image" src="https://github.com/internetarchive/openlibrary/assets/921217/29eefd04-264c-45bc-ae99-45bd7ffd1512">

**Comment by mekarpeles** [comment]
@RayBB let's make sure we get to a decision on a community call or design call please!

**Comment by RayBB** [comment]
FYI, I have updated the ticket to reflect the decision from the community call.

**Comment by RayBB** [comment]
@Barbara0608 yes go for it! Please let me know if you have any questions :)

**Comment by Barbara0608** [comment]
Hi @RayBB!! I believe it's fixed. Let me know if I can make the PR so that I can change the wiki as well!

**Comment by RayBB** [comment]
@Barbara0608 it looks like you pushed your change to your fork of open library. To get it fixed on the main open library you'll need to open a pull request against this repository. I don't have it handy but the info should be in the contributor guidelines :)

Please include screenshots in your PR!

**Comment by RayBB** [comment]
@Barbara0608 are you ready to open a pull request for your changes?

**Comment by RayBB** [comment]
@Barbara0608 looks like you're on the right track. For me to review the changes you'll need to open a PR in the internetarchive/openlibrary repo. You should be able to following these steps: https://github.com/internetarchive/openlibrary/wiki/Git-Cheat-Sheet#creating-a-pull-request
